cassandra/src/main/java/com/yahoo/ycsb/db/CassandraCQLClient.java

```diff
@@ -287,7 +330,7 @@ public Status read(String table, String key, Set<String> fields,
 
     } catch (Exception e) {
       e.printStackTrace();
```

Comment:
this should be replaced by passing the exception as a final argument to the logger call below